### DISCUSSION TITLE
Submitting an authority that is a new edition of an old source

### DISCUSSION THREAD

**Discussion by ptgolden** [periodo/periodo-data#83] [Bibliographic description]
@atomrab, I have a question for you. Since we're winding down development work, I wanted to submit some periods from books I have laying around the house. One is Lewis Mumford's Technics & Civilization, which was published in 1934, but which I have a [2010 publication of](https://press.uchicago.edu/ucp/books/book/chicago/T/bo10388066.html). How should I fill in this authority?

Adding the [WorldCat link](https://www.worldcat.org/title/technics-and-civilization/oclc/868281404) sets the year published as 2010. While that's technically correct, it's obviously not what we want to have in our data, because the periods were asserted in 1934. I could add a WorldCat link to the 1934 first edition as the authority, but I don't have access to that edition, and the page numbers would be incorrect.

My hunch is to set the year published to 1934, and then write out a full text citation indicating the edition that I have. What are your thoughts?

**Comment by ptgolden** [comment]
It actually looks like this edition preserves the original page numbering, but I'm still curious what you would say when that is not the case

**Comment by atomrab** [comment]
I ran into this in a big way with the Priestley chart, which was published in many, many versions, some with significant differences (see my editorial notes at http://n2t.net/ark:/99152/p05hrsf). My own preference is to get back to the original publication wherever possible, especially when we're dealing with reprints rather than revised editions. I would much rather have the older version represented, for obvious reasons, and if the pagination is the same I think you can use the 1934 WorldCat reference with an editorial note that you consulted it in an identical reprint of 2010. It's a little sneaky, but I don't think problematic.

But if the pagination is different, I think we'd be introducing potential error and eroding our commitment to transparency in sourcing. A user might try to track down the reference in the early edition and then not find it. So if there were no way to get to the 1934 version, I would say you'd have to use the 2010 as is, with a note that it was first published in 1934, and then we could try to get the earlier version and add it (or update the existing collection, if it was only a matter of a pagination change). @rybesh, what do you think?

**Comment by rybesh** [comment]
I guess I would enter the earliest publication details as the main bibliographic info for the authority, but if the page numbers refer to a later edition, then note that in the editorial note for the authority and put a citation to the later edition in the note too.

**Comment by atomrab** [comment]
I'm comfortable with that, as long as we're dealing essentially with reprints rather than editions with significant updates. But I do think that for later editions with substantial revisions, it's better to have both the older and newer versions, as we did with the Norton anthology. We don't want to attribute a definition that was substantially changed for the 2010 edition to the 1934 edition; this will just muddy the waters for the network of influence we're trying to document.

**Comment by rybesh** [comment]
Yes, I agree.
